application/helpers/expressions/em_manager_helper.php

```diff
@@ -4943,10 +4943,6 @@ static function StartSurvey($surveyid,$surveyMode='group',$aSurveyOptions=NULL,$
                     {
                         $knownVar = $LEM->knownVars[$LEM->qcode2sgqa[$k]];
                     }
-                    else if (isset($LEM->tempVars[$k]))
```

Comment:
Here it's for "prefill answer by survey". and $tempVars is not used for prefilling answers.